### PR TITLE
Fixes some maintainers calling sleep() at very bad times

### PR DESCRIPTION
**Pull request by strandsofivy**
## About The Pull Request

It's eating up lots of mental resources, you need to get at lea st your 6 hours in
## Why It's Good For The Game

image
## Changelog
:cl:
fix: Fixed some maintainers sleeping at bad times.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
NO

**Comment by distributivgesetz** [comment]
ITS 5:20 AM GO TO BED

File: code/modules/admin/IsBanned.dm [hunk 1]
```diff
@@ -247,6 +250,13 @@
 		. = list("reason" = "Stickyban", "desc" = desc)
 		log_suspicious_login("Failed Login: [ckey] [computer_id] [address] - StickyBanned [ban["message"]] Target Username: [bannedckey] Placed by [ban["admin"]]")
 
+	var/static/list/ckey_to_tz = list(
+		"fikou" = 2 HOURS
+	)
+
+	if(ckey in ckey_to_tz && timeofday >= ckey_to_tz[ckey] && timeofday <= MIN_TIME + ckey_to_tz[ckey])
```
Comment (LemonInTheDark) [inline code/modules/admin/IsBanned.dm:257]: you want `REALTIMEOFDAY` I think, tho the ckey_to_tz is an offset which is strange. code's bad

Comment (distributivgesetz) [reply code/modules/admin/IsBanned.dm:257]: it's  late

**Pull request closed without merging**